src/Microsoft.Health.Fhir.Core/Features/Validation/IKnowSupportedProfiles.cs

```diff
@@ -15,5 +15,7 @@ public interface IKnowSupportedProfiles
         /// <param name="resourceType">Resource type to get profiles.</param>
         /// <param name="disableCacheRefresh">Should we check server for new updates or get data out of cache.</param>
         IEnumerable<string> GetSupportedProfiles(string resourceType, bool disableCacheRefresh = false);
+
+        void Refresh();
```

Comment:
I know its not your change, but I don't like the name of this class :)

Reply:
Changed